docs/usage.rst

```diff
@@ -0,0 +1,280 @@
+   * Actual (Alternative): Use a custom-made series (not accounting moves) with at
+     least following fields: initial, debit, credit, ending, dates
+   * Sum Columns: Creates a new column sum of others existing in the report
+   * Compare Columns: Creates a comparison (percentage) of 2 existing columns
+   * MIS Budget (If budget installed): Use Budget Series
+#. Dates Mode:
+
+   * Fixed Dates: simply provide the fixed range
+   * Relative to report base date: allows to select the period compared with base report date.
+
+     *  `Offset = -1` means previous month.
+     *  `Duration = 2` means 2 months from the start of the period
+
+Manage your Budget
+******************
+Menu accessible here: `Accounting > Reports > MIS Reports > MIS Budget`
+
+To use this module, you first need to flag at least one KPI in a MIS Report to be
+budgetable. You also need to configure the accumulation method on the KPI according
+to their type.
+
+The accumulation method determines how budgeted values spanning over a time period
+are transformed to match the reporting period.
+
+* **Sum**: values of shorter period are added, values of longest or partially overlapping
+  periods are adjusted pro-rata temporis (eg monetary amount such as revenue).
+* **Average**: values of included period are averaged with a pro-rata temporis weight.
+  Typically used for values that do not accumulate over type (eg a number of employees).
+
+When KPI are configured, you need to:
+
+#. Create a budget, then
+#. Click on the budget items button to create or import the budgeted amounts for all your KPI
+   and time periods.
+#. Finally, a column (aka period) must be added to a MIS report instance, selecting your
+   newly created budget as a data source.
+#. The data will be adjusted to the reporting period when displayed.
+#. Columns can be compared by adding a column of type "comparison" or "sum".
+
+.. image:: images/13.png
+   :width: 1800
+
+.. image:: images/04.png
+   :width: 1800
+
+Create a new budget
+###################
+A budget is linked to a set of KPI created in the Report Template. A budget must be
+defined for a specific period (year or month)
+
+#. Provide a Name and Description
+#. Provide the Template it applies to
+#. Provide the date range. This is important because if a year is given, when the report
+   must display 3 months, current figures will be provided prorata temporis.
+#. Save and provide the budget Items
+
+.. image:: images/05.png
+   :width: 1800
+
+Fill in your Budget items
+#########################
+A budget contains "Budget items" (smart button on top right) which are the budget values
+for a KPI and a specific data.
+
+#. Select the KPI to be budgeted
+#. Select the date range or From/To
+#. provide Amount in Company currency
+#. Provide the corresponding Analytic account
+
+.. note:: To be noted that the budget could be prepared in Excel and easily imported via Odoo Standard Function.
+
+   .. image:: images/14.png
+      :width: 1800
```

Comment:
Note for this screenshot you can keep only the following columns:
* kpi_expression_id: with the kpi name or kpi_name.subkpi_name in presence of subkpi (no need to lookup kpi xmlids, the name search will do its magic as it knows the budget context)
* date_from
* date_to
* amount
* analytic account

The budget_id column is defaulted from the context when in the budget item views. And the date range is just a UI helper to enter date from/to.